api_test.go

```diff
@@ -1237,14 +1239,6 @@ var _ = Describe("RestHandler", func() {
 				"data": []interface{}{post1JSON},
 			}))
 		})
-
-		It("Extracts multiple parameters correctly", func() {
-			req, err := http.NewRequest("GET", "/v0/posts?sort=title,date", nil)
```

Comment:
The reason I removed is because buildRequest depends on the Resource now, because it calls unmarshalRequest() which needs res.marshalers.

I'll find another way to test it.